package.json

```diff
@@ -11,7 +11,11 @@
 		"url": "https://sindresorhus.com"
 	},
 	"type": "module",
-	"exports": "./index.js",
+	"exports": {
+		"types": "./index.js",
```

Comment:
```suggestion
		"types": "./index.d.ts",
```

Reply:
Originally used `public-ip` as a boilerplate. See https://github.com/sindresorhus/public-ip/pull/64 for a fix there as well.